corehq/apps/case_search/utils.py

```diff
@@ -116,7 +136,12 @@ def _add_case_property_queries(self):
             )
             for removal_regex in remove_char_regexs:
                 to_remove = re.escape(removal_regex.regex)
-                value = re.sub(to_remove, '', value)
+                if isinstance(value, list):
+                    new_value = []
+                    for val in value:
+                        new_value.append(re.sub(to_remove, '', value))
```

Comment:
* sub should happen on `val`, not `value`. 
* `new_value` needs to be assigned back to `value`

You could also do this more simply:
```suggestion
                    value = [re.sub(to_remove, '', val) for val in value]
```

Reply:
Thanks!